tasks/store.py

```
@@ -57,6 +57,30 @@ class TaskStore:
+
+        backing are unreachable from the dashboard AND from the bot callback
+
+        handler, which makes the `requires_approval` gate hostile to end-users.
+
+        Without this guard, a worker process whose DB connection fails
+
+        silently queues notifications referencing phantom ids.
+
+        """
+
+        if db is None and not os.environ.get("TESTING", "").strip().lower() in ("1", "true", "yes"):
```

Ruff (0.15.21)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)